### PR TITLE
fix errors in accessibility test #218

### PR DESCRIPTION
**Pull request by oburasamuel**
Fixed act warnings appearing in the accessibility tests, and one test was missing the view prop

also added a test for compare view in dark mode.

### CODE REVIEW

**Review by kimberlythegeek — APPROVED** [review]
awesome, thank you!

**Comment by oburasamuel** [comment]
Thank you for your feedback

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ PerfCompare is hosted on heroku, and is updated every time commits are pushed to
 ## Setup
 
 ### Requirements
-
```
Comment (oburasamuel) [inline README.md]: Fixed act warnings appearing in the accessibility tests, 
Fixed one test was missing the view prop
also added a test for compare view in dark mode.

File: src/__tests__/accessiblity/accessibility.test.tsx [hunk 2]
```diff
@@ -14,11 +14,11 @@ import { renderWithRouter, store } from '../utils/setupTests';
 expect.extend(toHaveNoViolations);
 
 describe('Accessibility', () => {
-  beforeEach(() => {
+  beforeAll(() => {
```
Comment (taofeeqhamzat) [inline src/__tests__/accessiblity/accessibility.test.tsx:17]: I don't get this, @oburasamuel. Could you explain why this change?
As a result of using `beforeAll()`, running the accessibility test (`test:a11y`) now times out.
